COVID_19_model.ipynb

```diff
@@ -1573,11 +1592,13 @@
         "    # plotly_offline_nojs(fig,\n",
         "    #                     filename='forecast_plots/no_js-' + plot_title + (\n",
         "    #                         plot_extension) + '.html')\n",
+        "  \n",
+        "  return traces\n",
         "\n",
         "forecast_data = generate_country_forecast(\n",
         "    MAP_params, map_estimate, param_summary, simple_model_data\n",
         ")\n",
-        "plot_country_forecast(studied_country, forecast_data)"
+        "plot_country_forecast(studied_country, forecast_data);"
```

Comment:
Does the semi colon serve a purpose?

Reply:
Yes, the function now returns all traces and it's messy without suppressing the output

Reply:
Ah, didn't know you could do that.